third-party/hwloc/README

```diff
@@ -18,16 +18,25 @@ The directory $CHPL_HOME/third-party/hwloc/hwloc-src/ contains the
 un-tarballed hwloc package contents. Version updates should be done as
 follows, assuming the CWD is $CHPL_HOME/third-party/hwloc/:
 
-1. download and untar the latest hwloc 1.X version: e.g. hwloc-1.11.13
+1. download and untar the latest hwloc 2.X version: e.g. hwloc-2.9.0
 2. `rm -rf hwloc-src`
-3. `mv hwloc-1.11.13 hwloc-src`
+3. `mv hwloc-2.9.0 hwloc-src`
 4. `git add --force hwloc-src` (--force to ignore our .gitignore)
 5. update the version number mentioned above
 6. update the okversions whitelist in chpl_hwloc.py
 7. make sure these instructions are up to date :)
 8. test (std config, and that stream gets expected performance)
 9. commit, PR, merge, etc
 
-Note that these instructions are for simple 1.X API compatible updates.
+Note that these instructions are for simple 2.X API compatible updates.
 If the hwloc API changes, the runtime shim may need to be changed and
 incompatible versions need to be dropped from the whitelist.
+
+Chapel modifications to hwloc
+================================
+
+We made the following modifications to the official hwloc release:
+
+* Add HWLOC_DARWIN_LDFLAGS to Libs.private in hwloc.pc.in so that
+Chapel programs link correctly on a Mac.
```

Comment:
Please open an hwloc issue for this (ideally a standalone reproducer that doesn't invovle chapel)

Reply:
I've created an issue (https://github.com/Cray/chapel-private/issues/5102) to do this.